### DISCUSSION TITLE
problem importing metran

### DISCUSSION THREAD

**Discussion by msh855** [pastas/metran#17] [Q&A]
Hi 

I am getting this error when trying to load the package 

`AttributeError: module 'pastas' has no attribute 'stats'`

**Comment by raoulcollenteur** [comment]
have you installed Pastas?

**Comment by msh855** [comment]
yes

**Comment by raoulcollenteur** [comment]
> Which version of pastas are you on?
> 
> ```python
> import pastas as ps
> 
> ps. __version__
> ```
> 
> Perhaps try upgrading?
> 
> `pip install pastas --upgrade`


I think @dbrakenhoff  meant to post this here.

**Comment by msh855** [comment]
No, luck. Problem still persists. I think might be dependency issue with matplotlib

**Comment by msh855** [comment]
pastas 0.22 version

**Comment by martinvonk** [comment]
Maybe you can create a new virtual environment, install Metran + Pastas and check if the issue persists?
